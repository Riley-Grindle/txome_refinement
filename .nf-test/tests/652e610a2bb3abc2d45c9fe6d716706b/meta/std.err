[33mNextflow 24.10.2 is available - Please consider updating your version to it(B[m
