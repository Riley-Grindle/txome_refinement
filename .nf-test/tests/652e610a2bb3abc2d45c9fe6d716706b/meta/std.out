ERROR ~ Error executing process > 'AGAT_CONVERTSPGXF2GXF (TEST)'

Caused by:
  Process `AGAT_CONVERTSPGXF2GXF (TEST)` terminated with an error exit status (127)


Command executed:

  agat config --expose
  
  sed -i.bak 's/output_format: *.*/output_format: GTF/' agat_config.yaml
  rm *.bak
  
  agat_convert_sp_gxf2gxf.pl \
      -g Danio_rerio.GRCz11.110.chr1.gtf.gz \
      -o TEST.agat.gtf \
  
  
  cat <<-END_VERSIONS > versions.yml
  "AGAT_CONVERTSPGXF2GXF":
      agat: $(agat_convert_sp_gxf2gxf.pl --help | sed '4!d; s/.*v//')
  END_VERSIONS

Command exit status:
  127

Command output:
  (empty)

Command error:
  .command.run: line 301: docker: command not found

Work dir:
  /mnt/jfs/nextflow/txome_refinement/.nf-test/tests/652e610a2bb3abc2d45c9fe6d716706b/work/48/78dbbc4932dacb16cd844b4fa58e53

Container:
  quay.io/biocontainers/agat:1.4.0--pl5321hdfd78af_0

Tip: you can try to figure out what's wrong by changing to the process work dir and showing the script file named `.command.sh`

 -- Check '/mnt/jfs/nextflow/txome_refinement/.nf-test/tests/652e610a2bb3abc2d45c9fe6d716706b/meta/nextflow.log' file for details
